src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const employeesArray = document.querySelectorAll('li');
+const employeesList = [];
+const ulElement = document.querySelector('ul');
+
+// Перебираем элементы и добавляем объекты в массив
+employeesArray.forEach((employee) => {
+  employeesList.push({
+    name: employee.textContent.trim(),
+    position: employee.dataset.position,
+    salary: Number(employee.dataset.salary.replace('$', '').replace(',', '')),
```

Comment:
Consider using a more robust method to convert the salary string to a number, such as using a regular expression to remove all non-numeric characters. This will ensure that any additional formatting in the salary string won't cause issues.